Lovertab/timesavor
# 7 make recipe api request for the chosen time slot

## Purpose

Please provide a detailed explanation of the PR. What existing problem does it solve?
added functions to fetch data from the api
added functions to display data from the api

Fixes #[7]

## Changes

Please provide an overview of code changes, especially key files and areas of significant change.

assets/js/script.js

## Steps to Reproduce or Test

Demonstrate that the pull request is functional. For example: the exact commands to be run and their output; screenshots if the PR modifies the UI.

- pull the branch
-  run index.html in live server
- check console logs to see api data

## Optional GIF
![CleanShot 2024-03-19 at 13 32 40@2x](https://github.com/Lovertab/timesavor/assets/66334129/2a8f4b69-0ff6-4b50-b23c-5e0f753b8df1)


Self-explanatory.
